src/main/mdo/auditor-results-model.mdo

```diff
@@ -27,6 +27,13 @@
       <description>Encapsulator of all results from an audit report run, and their HTML table header.</description>
       <version>1.0.0+</version>
       <fields>
```

Comment:
I'd prolly add an "id" field that's either random or computed.  That way the (final artifact xml) could be aggregated

Reply:
ALSO:
I don't see a timestamp in here anywhere

Reply:
Add a `.gitignore` 

```
.*
!.gitignore
target
*.iml
pom.xml.*
release.properties
/bin
```

Reply:
Also, don't `    <class xml.tagName="AuditResults"`

Remove the `xml.tagName` as unnecesary.  It'll make the result `auditResults` (I think)